EED_Multi_Event_Registration.module.php

```diff
@@ -1172,15 +1173,16 @@ function( $additional_info ) {
 	 *    adjust_ticket_quantity
 	 *
 	 * @access    protected
-	 * @param EE_Ticket $ticket
-	 * @param int       $quantity
-	 * @param bool      $cart_update
+	 * @param EE_Ticket    $ticket
+	 * @param int          $quantity
+	 * @param EE_Line_Item $line_item
+	 * @param string       $action
 	 * @return bool
 	 */
-	protected function can_purchase_ticket_quantity( $ticket = null, $quantity = 1, $cart_update = false ) {
+	protected function can_purchase_ticket_quantity( $ticket = null, $quantity = 1, $line_item = null, $action = 'add' ) {
 		// any tickets left at all?
 		$tickets_remaining = $ticket instanceof EE_Ticket ? $ticket->remaining() : 0;
-		if ( ! $tickets_remaining ) {
+		if ( ! $tickets_remaining && $action != 'remove') {
```

Comment:
we also try to always use type safe comparison 

```suggestion
		if ( ! $tickets_remaining && $action !== 'remove') {
```